graphql/resolvers/_mutations/commit.js

```diff
@@ -74,7 +74,7 @@ module.exports = async (_, args, { pgdb, req, user, t, pubsub }) => {
     })
     // auto_init seems to be async and takes some time
     // we sporadically saw: Git Repository is empty. on commit
-    await sleep(1000)
+    await sleep(3000)
```

Comment:
Off topic: could we change this to a api call that verifies this every 500ms? With a fail timeout of ~30s?

Quick fix: only increase to 3000ms in test env.

Reply:
thank's for the hint! I implemented polling. timeout: 6s should be more than enough IMHO.